spacepy/ae9ap9.py

```diff
@@ -529,9 +530,20 @@ def _parseInfo(header):
         elif "Energy levels" in val:
             match = re.match(r'^Energy levels.*\((.*)\):(.*)$', val)
             ans['energy'] = (np.asarray(match.group(2).strip().split()).astype(float), match.group(1).strip())
+        # Get the orbital element propagator, two versions based on AE9 model changes
+        # new format data file
+        elif "Propagator" in val:
+            match = re.search(r'Propagator:\ (.*)$', val)
+            ans['propagator'] = match.group(1).strip()
+        # old format data file
         elif "generated from specified elements" in val:
             match = re.search(r'^generated from specified elements.*:\ (.*)$', val)
-            ans['propagator'] = match.group(1).strip()
+            if match:
```

Comment:
I think this is actually ok, those words are in both old and new files. For the ephem files this catches old and new and for non ephem files falls thru as desired.

Reply:
I'm not asking about the if statement on (new) line 539), but line 541. The `if match` is new.

Reply:
I'm not totally sure I follow. For a new style file line 536 will be true and 541 will be false. The if allows new style processing without bombing on old. For old style the if on 535 is false then 541 will be true. Make sense?

Reply:
I pulled to have latest line numbers

```python
        # new format data file
        elif "Propagator" in val: # Line 535, true on new format
            match = re.search(r'Propagator:\ (.*)$', val)
            ans['propagator'] = match.group(1).strip()
        # old format data file
        elif "generated from specified elements" in val: #line 539, true on old format
            match = re.search(r'^generated from specified elements.*:\ (.*)$', val)
            if match: #line 541, only reached on old format, but this is a test that was not in old code
```
For old style the test on line 539 is True. In what case do you expect that test to be true but the test on line 541 to be False, such that you added that test, which wasn't present in the original code to read the old file? And in that case is silently ignoring the line the correct thing to do, since that's what the code does now?